tracker/gioui/preferences.go

```diff
@@ -0,0 +1,65 @@
+package gioui
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+
+	"gioui.org/unit"
+)
+
+type (
+	Preferences struct {
+		Window   WindowPreferences
+		YmlError error
+	}
+
+	WindowPreferences struct {
+		Width     int
+		Height    int
+		Maximized bool `yaml:",omitempty"`
```

Comment:
nit: To me, "maximized" means a different concept from fullscreen. Maximized means the window takes maximal amount of space, but keeping the decoration. Fullscreen means the bottom bars and window decorations are gone.

Reply:
Gioui seems to agree: maximized, minimized, windowed and fullscreen are different options: https://github.com/gioui/gio/blob/8daff13af6cdbe4cb689e324f6ba343062dad050/app/window.go#L853

Reply:
Well, yeah, this confuses me. All I ever meant was "Maximized" (and I also see no use for actual Fullscreen), but when I try the gioui options, the app.Fullscreen.Option() that I used and the app.Maximized.Option() give the same effect (i.e. Maximized).

as the app.Fullscreen.Option() gave me what I wanted, I didn't get the idea of looking further; but as no other people really asked for Fullscreen, I would then just use the different Option, even if they look the same for me.